scripts/test/Muon/elemental_analysis/periodic_table_item_test.py

```diff
@@ -0,0 +1,72 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+from __future__ import print_function, absolute_import
+
+
+import unittest
+
+from mantid.py3compat import mock
+
+import Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table as periodic_table
+from Muon.GUI.ElementalAnalysis.PeriodicTable.periodic_table import PeriodicTableItem,\
+    ColoredPeriodicTableItem
+
+
+class PeriodicTableItemTest(unittest.TestCase):
+    def setUp(self):
+        self.element = PeriodicTableItem("Ti", 22, 4, 4, "titanium", 47.9000, "transition metal")
+
+    def test_that_list_elements_contains_all(self):
+        for i in range(len(periodic_table._elements)-1):
+            self.assertTrue(isinstance(periodic_table._elements[i][0], str))
+            self.assertTrue(isinstance(periodic_table._elements[i][1], int))
+            self.assertTrue(isinstance(periodic_table._elements[i][2], int))
+            self.assertTrue(isinstance(periodic_table._elements[i][3], int))
+            self.assertTrue(isinstance(periodic_table._elements[i][4], str))
+            self.assertTrue(isinstance(periodic_table._elements[i][5], float)
```

Comment:
why does this one have an or?

Reply:
For some elements the atomic weight is only approximate and represented as an integer, for most as a float.